packages/ethash/src/index.ts

```diff
@@ -143,14 +142,16 @@ export class Miner {
 
 export default class Ethash {
   dbOpts: Object
-  cacheDB?: LevelUp
+  // TODO: check what the correct type for a value is
+  cacheDB?: Level<Buffer, any>
   cache: Buffer[]
   epoc?: number
   fullSize?: number
   cacheSize?: number
   seed?: Buffer
 
-  constructor(cacheDB?: LevelUp) {
+  // TODO: check what the correct type for a value is
```

Comment:
I'll respond here for all the questions about the `Level<any,any>` typing question. Is it more trouble than it's worth to type it according to how we use keys and values in the DB?  I'm asking because I don't know but we allow `string | Buffer` today in keys and `Buffer | object` in values following [this](https://github.com/ethereumjs/ethereumjs-monorepo/blob/8a65e1928bc555519d22d3a5d5f885a54eaf0eff/packages/blockchain/src/db/operation.ts#L32).  If you define the db as `Level<string | Buffer, Buffer | object>` do you find any issues?

Reply:
Okay, that doesn't feel terrible to me since it should be localized within the `DBManager` or the related places deep in the code where we do DB ops.  If we can keep the external interfaces that return values to the various consumers intact, I would still like to have the DB key and value types defined wherever possible.